doc/source/whatsnew/v1.2.0.rst

```diff
@@ -49,6 +49,42 @@ For example:
    buffer = io.BytesIO()
    data.to_csv(buffer, mode="w+b", encoding="utf-8", compression="gzip")
 
+Arithmetic with Timestamp and Timedelta-based Intervals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Arithmetic can now be performed on :class:`Interval` s having their left and right
+ends as :class:`Timestamp`s or :class:`Timedelta` s, like what would be possible
+if the ends were numeric (:issue:`35908`).
+Before this change, it could be performed in an indirect way like this:
+
+.. ipython:: python
+
+   interval = pd.Interval(pd.Timestamp("1900-01-01"), pd.Timestamp("1900-01-02"))
+   pd.Interval(interval.left - pd.Timestamp("1900-01-01"), interval.right - pd.Timstamp("1900-01-01"))
+
+Now, this can be performed by directly using arithmetic operators (`-` or `+`):
```

Comment:
Have just one example. We'll say at the top that this works for both Timestamps and Timedeltas